zerver/views/webhooks/appfollow.py

```diff
@@ -0,0 +1,40 @@
+# Webhooks for external integrations.
+from __future__ import absolute_import
+import re
+
+from django.http import HttpRequest, HttpResponse
+
+from zerver.lib.actions import check_send_message
+from zerver.lib.response import json_success, json_error
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+from zerver.models import UserProfile, Client
+
+from typing import Text
+
+@api_key_only_webhook_view("AppFollow")
+@has_request_variables
+def api_appfollow_webhook(request, user_profile, client, stream=REQ(default="reviews"),
+                          payload=REQ(argument_type="body")):
+    # type: (HttpRequest, UserProfile, Client, Text, Text) -> HttpResponse
+    try:
+        message = payload["text"]
+    except KeyError:
+        return json_error("Missing 'text' argument in JSON")
+    app_name = re.search('\A(.+)', message).group(0)
+
+    check_send_message(user_profile, client, "stream", [stream], app_name, convert_markdown(message))
+    return json_success()
+
+def convert_markdown(text):
+    # type: (Text) -> Text
+    # Converts Slack-style markdown to Zulip format
+    # Implemented mainly for AppFollow messages
+    # Not ready for general use as some edge-cases not handled
+    # Convert Bold
+    text = re.sub(r'(?:(?<=\s)|(?<=^))\*(.+?\S)\*(?=\s|$)', r'**\1**', text)
```

Comment:
Need a look-around here because `*` is a non-word, so `\b` won't work here.
`?:(?<=\s)|(?<=^))` is needed because Python only supports fixed-length look-behinds, so `(?<=\s|^)` won't work.